src/lxc/start.c

```diff
@@ -937,7 +937,7 @@ static int do_start(void *data)
 	}
 
 	/* Setup the container, ip, names, utsname, ... */
-	ret = lxc_setup(handler);
+	ret = lxc_setup_child(handler);
```

Comment:
This belongs in the previous patch, right?

Reply:
addressed